extensions/Khronos/KHR_materials_pbrSpecularGlossiness/README.md

```diff
@@ -0,0 +1,214 @@
+
+*c<sub>diff</sub>* = `diffuse.rgb * (1 - max(specular.r, specular.g, specular.b)`
+<br>
+*F<sub>0</sub>* = `specular`
+<br>
+*&alpha;* = `max((1 - glossiness) ^ 2, epsilon)`
+
+All implementations should use the same calculations for the BRDF inputs. Implementations of the BRDF itself can vary based on device performance and resource constraints. See [appendix](/specification/2.0/README.md#appendix-a) for more details on the BRDF calculations.
+
+The following table lists the allowed types and ranges for the specular-glossiness model:
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**diffuseFactor** | `number[4]` | The reflected diffuse factor of the material.|No, default:`[1.0,1.0,1.0,1.0]`|
+|**diffuseTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)  | The diffuse texture.|No|
+|**specularFactor** | `number[3]` | The specular RGB color of the material. |No, default:`[1.0,1.0,1.0]`|
+|**glossinessFactor** | `number` | The glossiness or smoothness of the material. |No, default:`1.0`|
+|**specularGlossinessTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | The specular-glossiness texture.|No|
+
+Additional properties are not allowed.
+
+* **JSON schema**: [glTF.KHR_materials_pbrSpecularGlossiness.schema.json](schema/glTF.KHR_materials_pbrSpecularGlossiness.schema.json)
+
+### diffuseFactor
+
+The RGBA components of the reflected diffuse color of the material. Metals have a diffuse value of `[0.0, 0.0, 0.0]`. The fourth component (A) is the opacity of the material. The values are linear.
+
+* **Type**: `number[4]`
+* **Required**: No, default:`[1.0,1.0,1.0,1.0]`
+* **Range**: [0,1] for all components
+
+### diffuseTexture
+
+The diffuse texture. This texture contains RGB(A) components of the reflected diffuse color of the material in sRGB color space. If the fourth component (A) is present, it represents the opacity of the material. Otherwise, an opacity of 1.0 is assumed.
+
+* **Type**: [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)  
+* **Required**: No
+
+### specularFactor
+
+The specular RGB color of the material. This value is linear.
+
+* **Type**: `number[3]`
+* **Required**: No, default:`[1.0,1.0,1.0]`
+* **Range**: [0,1] for all components
+
+
+### glossinessFactor
+
+The glossiness or smoothness of the material. A value of 1.0 means the material has full glossiness or is perfectly smooth. A value of 0.0 means the material has no glossiness or is perfectly rough. This value is linear.
+
+* **Type**: `number`
+* **Required**: No, default:`1.0`
+* **Range**: [0,1]
+
+### specularGlossinessTexture
+
+The specular-glossiness texture is a RGBA texture, containing the specular color (RGB) in sRGB space and the glossiness value (A) in linear space.
+
+* **Type**: [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) 
+* **Required**: No
+
+
+#### Additional maps
```

Comment:
My opinion is that texture is a type where map is a concept (i.e. maps are stored as textures). That said, maybe we can call this "common properties" to avoid the problem all together.